docset/winserver2012-ps/bitlocker/Enable-BitLocker.md

```diff
@@ -217,308 +270,312 @@ Accept wildcard characters: False
-To obtain a BitLocker volume object, use the Get-BitLockerVolume cmdlet.
+To obtain a BitLocker volume object, use the **Get-BitLockerVolume** cmdlet.
 
 ```yaml
 Type: String[]
-Parameter Sets: UNNAMED_PARAMETER_SET_2
+Parameter Sets: (All)
 Aliases: 
 
 Required: True
-Position: 1
+Position: 0
 Default value: None
-Accept pipeline input: True (ByValue, ByPropertyName)
+Accept pipeline input: True (ByPropertyName, ByValue)
 Accept wildcard characters: False
 ```
 
 ### -Password
+
 Specifies a secure string object that contains a password.
 The password specified acts as a protector for the volume encryption key.
 
 ```yaml
 Type: SecureString
-Parameter Sets: UNNAMED_PARAMETER_SET_3
+Parameter Sets: PasswordProtector
 Aliases: pw
 
 Required: False
-Position: 2
+Position: 1
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -PasswordProtector
+
 Indicates that BitLocker uses a password as a protector for the volume encryption key.
 
 ```yaml
 Type: SwitchParameter
-Parameter Sets: UNNAMED_PARAMETER_SET_3
+Parameter Sets: PasswordProtector
 Aliases: pwp
 
 Required: True
 Position: Named
-Default value: False
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -Pin
+
 Specifies a secure string object that contains a PIN.
 BitLocker uses the PIN specified, with other data, as a protector for the volume encryption key.
 
 ```yaml
 Type: SecureString
-Parameter Sets: UNNAMED_PARAMETER_SET_4, UNNAMED_PARAMETER_SET_5
+Parameter Sets: TpmAndPinAndStartupKeyProtector, TpmAndPinProtector
 Aliases: p
 
 Required: False
-Position: 2
+Position: 1
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -RecoveryKeyPath
+
 Specifies a path to a folder.
 This cmdlet adds a randomly generated recovery key as a protector for the volume encryption key and stores it in the specified path.
 
 ```yaml
 Type: String
-Parameter Sets: UNNAMED_PARAMETER_SET_6
+Parameter Sets: RecoveryKeyProtector
 Aliases: rk
 
 Required: True
-Position: 2
+Position: 1
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -RecoveryKeyProtector
+
 Indicates that BitLocker uses a recovery key as a protector for the volume encryption key.
 
 ```yaml
 Type: SwitchParameter
-Parameter Sets: UNNAMED_PARAMETER_SET_6
+Parameter Sets: RecoveryKeyProtector
 Aliases: rkp
 
 Required: True
 Position: Named
-Default value: False
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -RecoveryPassword
+
 Specifies a recovery password.
-If you do not specify this parameter, but you do include the **RecoveryPasswordProtector** parameter, the cmdlet creates a random password.
-You can enter a 48 digit password.
+If you do not specify this parameter, but you do include the *RecoveryPasswordProtector* parameter, the cmdlet creates a random password.
+You can enter a 48-digit password.
```

Comment:
```suggestion
You can enter a 48-digit password.

```

Reply:
GitHub error: "This suggestion is invalid because no changes were made to the code."